server/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:latest
+RUN DEBIAN_FRONTEND=noninteractive apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -yq tmux git python-dev \
+          python-mysqldb build-essential libmysqlclient-dev python-pip \
+          python-numpy python-sklearn netcat-traditional libmysqlclient-dev \
```

Comment:
On the contrary, it is a living hell to get them from pip as they build them from source.